<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormatText</class>
 <widget class="QDialog" name="FormatText">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="dateGroup">
     <property name="title">
      <string>Date</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Date source format</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="dateSourceFormat"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Date format</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="dateFormat">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>MM/dd/yyyy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dd/MM/yyyy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MM-dd-yy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dd-MM-yy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MMM,dd yyyy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MMMM, dd yyyy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dd MMMM yyyy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dddd MMMM, dd yyyy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dddd dd MMMM yyyyy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MMMM-dd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dd-MMM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MMMM-yy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>hh:mm:ss</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>hh:mm am/pm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>hh:mm AM/PM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MM/dd/yyyy hh:mm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dd/MM/yyyy hh:mm</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="integerGroup">
     <property name="title">
      <string>Interger</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Number of digit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="intDigit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="floatGroup">
     <property name="title">
      <string>Float</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Number of decimal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="floatDecimal"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Example</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="value"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Formatted value</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="formatted">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FormatText</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormatText</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
